examples/complete/vpc.tf

```diff
@@ -28,7 +28,7 @@ module "vpc" {
   public_subnets_cidr_blocks         = var.public_subnets_cidr_blocks
   # Create subnets flag
   create_public_subnets         = true
-  enable_nat_gateway            = true
+  enable_nat_gateway            = false
```

Comment:
@souza-dan the value was changed to true

Reply:
Was this example working before this PR when enable_nat_gateway was set to true?

Reply:
Yes, it was working then and is working now, but with the changes on the emr bootstrap actions. This PR set the enable_nat_gateway to false from the beginning.

Reply:
What's the impact of having the NAT gateway enabled vs disabled?

I'm not seeing the benefit of keeping this false, which doesn't justify having messier bootstrap actions.

Reply:
**NAT gateway enabled:**
- We can use the S3 URI or AWS CLI; it will work in both cases on regions different from us-east-1 

**NAT gateway disabled** 
- S3 URI won't work in regions different from us-east-1

With the changes on the bootstrap actions to the AWS CLI, the complete example works with the NAT gateway enabled or disabled. Without these changes, it will only work with the NAT gateway enabled.